st2common/tests/unit/test_util_sandboxing.py

```diff
@@ -9,15 +9,34 @@
 from st2common.constants.pack import SYSTEM_PACK_NAMES
 from st2common.util.sandboxing import get_sandbox_path
 from st2common.util.sandboxing import get_sandbox_python_path
+from st2common.util.sandboxing import get_sandbox_python_path_for_python_action
 from st2common.util.sandboxing import get_sandbox_python_binary_path
+
 import st2tests.config as tests_config
 
+__all__ = [
+    'SandboxingUtilsTestCase'
+]
+
 
 class SandboxingUtilsTestCase(unittest.TestCase):
+    def setUp(self):
+        super(SandboxingUtilsTestCase, self).setUp()
+
+        # Restore PATH and other variables before each test case
+        os.environ['PATH'] = self.old_path
+        os.environ['PYTHONPATH'] = self.old_python_path
+        sys.real_prefix = self.old_real_prefix
+
     @classmethod
     def setUpClass(cls):
         tests_config.parse_args()
 
+        # Store original values so we can restore them in setUp
+        cls.old_path = os.environ.get('PATH', '')
+        cls.old_python_path = os.environ.get('PYTHONPATH', '')
+        cls.old_real_prefix = sys.real_prefix
```

Comment:
Should you restore them in `tearDownClass`? If not it seems like it might affect other tests.

Reply:
I restore it in `setUp` which runs before every test case run, but yeah, it's also a good idea to restore it in `tearDownClass` to avoid cross test file pollution :+1: